docs/learn/learn-runtime-upgrades.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: learn-runtime-upgrades
 title: Runtime Upgrades
-sidebar_label: Runtime Upgrades
+sidebar_label: Upgrades
```

Comment:
Upgrades is too generic. Runtime upgrades are different from node/client upgrades 

```suggestion
sidebar_label: Runtime Upgrades
```